index.html

```diff
@@ -231,11 +231,13 @@
 
 # Thank you
 
-Meili
+Github: mei-li
 
-meili.tr@gmail.com
+Email: meili.tr@gmail.com
 
-@meilitr
+Twitter: @meilitr
+
+Thank you all you are lovely
```

Comment:
Thank you for sharing with us more about how we can improve our knowledge at the same time build our career.#Alexie_MANIRAKORA
Nice to meet you!